hazelcast-integration/spring-hibernate-2ndlevel-cache/pom.xml

```diff
@@ -28,35 +40,30 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
-			<exclusions>
-				<exclusion>
-					<groupId>org.junit.vintage</groupId>
-					<artifactId>junit-vintage-engine</artifactId>
-				</exclusion>
-			</exclusions>
 		</dependency>
 
 		<dependency>
 			<groupId>com.hazelcast</groupId>
 			<artifactId>hazelcast-hibernate53</artifactId>
-			<version>2.0.0</version>
+			<version>2.2.2-SNAPSHOT</version>
```

Comment:
We should update to `2.3.0` when we release.

Reply:
Yes, will send a PR after the publish

Reply:
actually added a commit here: https://github.com/hazelcast/hazelcast-code-samples/pull/550/commits/ec71093961414f289fe2e464dd48059f79b2f92b